src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/** 
+ * This class represents the coordinates of a site location.
+ */
+public class Coordinates {
+  @SerializedName(value = "Longitude")
+  private final double lng;
```

Comment:
Given the current arrangement of public/privates, using this object will be problematic.

Aside from toString(), there's no way for something outside the class to get access to the lat/lng this holds.

If this is a passive data structure, then it's members shop be public so they can be accessed.